api/internal/core/storage/etcd.go

```diff
@@ -32,11 +32,16 @@ var (
 type EtcdV3Storage struct {
 }
 
-func InitETCDClient(endpoints []string) error {
-	cli, err := clientv3.New(clientv3.Config{
+func InitETCDClient(endpoints []string, username, password string) error {
```

Comment:
if we set typing for username?

Reply:
username string,password string = username,password string